### PR TITLE
mongo-replica-config.service works on digitalocean

### PR DESCRIPTION
**Pull request by jaigouk**
fleetctl version 0.9.0
mongodb version 2.6.7
coreos version 522.5.0

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,12 +1,14 @@
...
 
 ## Deploy
 
 ```
+fleetctl destroy mongo@{1..3}.service  mongo-replica-config.service 
+
 etcdctl set /mongo/replica/name myreplica
-fleetctl start mongo@{1..3}.service mongo-replica-config.service
+
+fleetctl start mongo-data@{1..3}.service mongo@{1..3}.service mongo-replica-config.service
```
Comment (jfromaniello) [inline README.md]: where does this mongo-data comes from?

Comment (jaigouk) [inline README.md]: Oh. that was a typo. I forked this repo to other one to use volume container(https://github.com/19hz/coreos-mongodb-cluster). 

I deleted mongo-data@.service related lines from README.

**Comment by jfromaniello** [comment]
Looks awesome man! :)

**Comment by jfromaniello** [comment]
Thank you very much for this

**Pull request merged**